sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -221,8 +221,7 @@ object ScalaNativePluginInternal {
             val isCppSource = path.endsWith(".cpp")
 
             val compiler = abs(if (isCppSource) clangpp else clang)
-            val flags = Seq("-O2") ++ (if (isCppSource) Seq("-std=c++11")
-                                       else Seq()) ++ opts
+            val flags    = (if (isCppSource) Seq("-std=c++11") else Seq()) ++ opts
```

Comment:
This is not the case at the moment. Most of the C/C++ code we have out there are just simple wrappers without any logic whatsoever, there isn't anything to debug in there.